app/assets/javascripts/application.js

```diff
@@ -135,8 +135,9 @@ $(document).on('ready', function() {
     });
   });
 
-
-  $('.js-first-visit-notice').on('close.bs.alert', async () => {
-    document.cookie = 'dismiss_fvn=true; path=/; expires=Fri, 31 Dec 9999 23:59:59 GMT';
-  });
+  if (document.cookie.indexOf ('dismiss_fvn') == -1 ) {
+    document.getElementById("fvn-dismiss").onclick = function() {
+      document.cookie = 'dismiss_fvn=true; path=/; expires=Fri, 31 Dec 9999 23:59:59 GMT';
+    }
```

Comment:
**suggestion** (es6, libraries, blocking): we have both ES6 and jQuery available, which we should use to maintain consistency here:

```suggestion
    $('#fvn_dismiss').on('click', () => {
      document.cookie = 'dismiss_fvn=true; path=/; expires=Fri, 31 Dec 9999 23:59:59 GMT';
    });
```

Reply:
All changes in this comment implemented via committing of the suggested change